Mathlib/Analysis/Normed/Group/Seminorm.lean

```diff
@@ -488,8 +510,29 @@ theorem zero_apply (x : E) : (0 : NonarchAddGroupSeminorm E) x = 0 :=
 instance : Inhabited (NonarchAddGroupSeminorm E) :=
   ⟨0⟩
 
--- TODO: define `SupSet` too, from the skeleton at
--- https://github.com/leanprover-community/mathlib/pull/11329#issuecomment-1008915345
+open Classical in
+noncomputable instance : SupSet (NonarchAddGroupSeminorm E) :=
```

Comment:
The same comment apply to this one too. You need the rewriting lemmas, `where` is better syntax, and you can probably simplify the proof slightly.